backend/fleet_management/models.py

```diff
@@ -3,31 +3,31 @@
 from django.utils.timezone import now
 
 
-class DriverUserProfile(AbstractUser):
+class Driver(AbstractUser):
 
     def __str__(self):
         return self.username
 
 
-class CarModel(models.Model):
+class Car(models.Model):
     plates = models.CharField(max_length=10, blank=False, unique=True)
 
     def __str__(self):
         return self.plates
 
 
-class PassengerModel(models.Model):
-    first_name = models.CharField(max_length=20, blank=False, unique=True)
-    last_name = models.CharField(max_length=20, blank=False, unique=True)
+class Passenger(models.Model):
+    first_name = models.CharField(max_length=60, blank=False, unique=True)
+    last_name = models.CharField(max_length=60, blank=False, unique=True)
 
     def __str__(self):
         return self.last_name
 
 
-class RouteModel(models.Model):
-    driver = models.ForeignKey(DriverUserProfile, on_delete=models.CASCADE)
-    car = models.ForeignKey(CarModel, null=False, on_delete=models.CASCADE)
-    passengers = models.ManyToManyField(PassengerModel)
+class Route(models.Model):
+    driver = models.ForeignKey(Driver, on_delete=models.CASCADE)
+    car = models.ForeignKey(Car, null=False, on_delete=models.CASCADE)
```

Comment:
More of a product requirement: do we want to delete drivers and cars? I think "disabling" them would be better - we may need the data later.

Reply:
Most probably we will disable them, but Django since version 2.x requires `on_delete` parameter.